content/knowledge-white-label/white-label-apps-overview.md

```diff
@@ -0,0 +1,121 @@
+
+In order to write your white labeling scripts, familiarity with shell scripting will also be an advantage. You can find some samples for common tasks [here](./white-label-scripts/).
+
+## Flavors versus a white labeling workflow
+
+With flavors you create different versions of your app within the same project. They are typically used when you would like to create a handful of different versions of your app so you can create   a “development” build, a “qa” build and a “production” build. Each version might be configured to point to a different API endpoint, or have a different icon to show which version it is so it’s clear which app is being used when testing the app.
+
+In Xcode you add different build schemes which can be used to customise the project output. Similarly, in Android Studio you can create product build variants and create a separate folder for each flavor’s resources.
+
+If you only needed to white label a few versions of your app then you could use this approach to build different versions of your app. However, it’s not very scalable and creates an unnecessarily large project and makes it harder to automate the white label process.
+
+An automated white label workflow,  on the other hand, doesn’t require that your app has multiple schemes or flavors configured in your Xcode or Android project. When the app is built, a series of scripts are run which change the project’s properties and change things like icons, images, fonts, values in xml and plist files. In this way, a single base app can be used to create as many different versions as you need without having to add the customisations directly to the project.
+
+
+
+## Limitations of white label workflows
+
+Neither Apple nor Google provide APIs that allow you to add a new application programmatically in their stores. This means you have to manually add the first version of a new application in the App Store or Google Play. Once this has been done it’s possible to automatically publish updated app builds to the stores. Tasks such as updating screenshots and descriptions or other meta data can be updated using third party tools such as Fastlane.
+
+## Pricing for building white label apps
+
+There is no extra charge for setting up white label workflows with Codemagic. However, if you are white labeling many versions of your app you might want to add additional concurrencies so multiple versions can be built in parallel. 
+
+Codemagic’s **pay-as-you-go plan** lets you use up to three concurrencies, whereas **Annual** and **Enterprise** plans have three concurrencies by default and you can add additional concurrencies as your needs grow. 
+
+You can find detailed pricing information [here](https://docs.codemagic.io/billing/pricing/).
+
+## Parallel white label builds
+
+If you have a large number of white label apps to it can take a many hours to build and publish them all. Fortunately with Codemagic, it is possible to use multiple concurrencies to run builds in parallel. 
```

Comment:
```suggestion
If you have a large number of white label apps, it can take many hours to build and publish them all. Fortunately, with Codemagic it is possible to use multiple concurrencies to run builds in parallel. 
```